draft-ietf-masque-connect-udp-listen.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Proxying Listener UDP in HTTP"
-abbrev: "CONNECT-UDP Listen"
+title: "Proxying Binding UDP in HTTP"
```

Comment:
Should we say "Binding" or "Bound"?

Reply:
Either way makes sense, as binding is an adjective here. I don't lean strongly either way.

Reply:
Cool. I'd say I have a preference for Bound if they're the same to you